docs/readme.ptxt

```diff
@@ -150,8 +150,8 @@ Mind to re-start OpenTTD as it needs to re-read the files.
 4 License
 =========
 
-OpenMSX Music Replacement Set for OpenTTD Copyright (C) 2010 OpenMSX Authors
-(see below or in the source in themes.list) and is licensed under GPL v2.
+OpenMSX for OpenTTD Copyright (C) 2010 OpenMSX Authors (see below or in the
+source in themes.list) and is licensed under GPL v2.
```

Comment:
I'd suggest always putting the words "music set" after the name OpenMSX, since the name is also used for an emulator of MSX compatible computers.

Reply:
The copyright date should probably be changed to 2010-2021 too.

Reply:
i agree, it should be "OpenMSX Music Set"